docs/pages/setup/guides/docker.mdx

```diff
@@ -1,60 +1,143 @@
 ---
-title: Getting started with Teleport using Docker
-description: How to get started with Teleport Open Source Edition using Docker locally.
+title: How to Run Teleport Using Docker
+description: How to run Teleport using Docker.
 h1: Run Teleport using Docker
 ---
 
-This section will cover:
+This guide will explain how to run a container using one of Teleport's Docker
+images and execute commands on that container via Teleport's `tsh` client.
 
-- Getting started with a local Teleport using Docker.
-- Using Teleport with Teleport's native client, `tsh`.
+<Notice scope={["cloud"]} type="tip">
 
-<Details scope={["cloud"]} title="Teleport Cloud" scopeOnly={true} opened={true}>
-Teleport Cloud includes managed instances of Teleport's Auth Service and Proxy Service. Since all of Teleport's services are run from the same binary, you can use our Docker image to run other services (e.g., the Database Service or App Service), or explore the Auth and Proxy Services locally.
-</Details>
+Since all of Teleport's services are run from the same binary, you can
+use our Docker image to run Node services (e.g., the Database Service or App
+Service) or explore the Auth and Proxy Services locally.
+
+</Notice>
 
 ## Prerequisites
 
-- Teleport v(=teleport.version=) Open Source or Enterprise.
-- Docker v(=docker.version=) or later.
+<Tabs>
+<TabItem scope={["oss"]} label="Open Source">
+
+- Docker v(=docker.version=) or later. We currently only offer Docker images for
+  `x86_64` architectures.
+
+```code
+$ docker version
+# Client: Docker Engine - Community
+# Version:           (=docker.version=)
+```
+
+- The `tsh` client tool, which ships with the `teleport` binary. Visit [Download Teleport](/teleport/download/) to download the open source edition.
+
+</TabItem>
+<TabItem scope={["cloud", "enterprise"]} label="Commercial">
+
+- Docker v(=docker.version=) or later. We currently only offer Docker images for
+  `x86_64` architectures.
 
 ```code
 $ docker version
 # Client: Docker Engine - Community
 # Version:           (=docker.version=)
 ```
 
+- The Enterprise edition of the `tsh` client tool, which ships with the `teleport` binary. Visit the [customer portal](https://dashboard.gravitational.com/web/login) to download Teleport.
```

Comment:
There is no such thing as the enterprise edition of `tsh`. Only `tctl` and `teleport` have enterprise editions.